polaris.shopify.com/src/components/Markdown/components/DirectiveCard/index.tsx

```diff
@@ -23,24 +24,41 @@ export type DirectiveStatus = DirectiveStatusName;
 type DirectiveProps = React.PropsWithChildren<{
   status: DirectiveStatus;
   img?: string;
+  alt?: string;
 }>;
-export const DirectiveCard = ({children, status, img}: DirectiveProps) => {
+export const DirectiveCard = ({children, status}: DirectiveProps) => {
+  const childrenArray = Children.toArray(children) as ReactElement[];
+
+  let image: ReactElement | undefined;
+
+  const rest = childrenArray.filter((d) => {
+    if (d.props.src) {
+      image = d;
```

Comment:
pulling out the image from the childrens array

Reply:
@yurm04 While I appreciate the impetus for this change, automatically making the latest image in children the feature image seems like something that might cause indirection and unintended bugs in the future. 

Can we just lean on explicitly passing the image and alt text to the Do / Don't component instead (since we're leaning on MDX anyways)